eli5/formatters/features.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+
+class EscapedFeatureName(object):
```

Comment:
What do you think about renaming it to something like FormattedFeatureName or PreparedFeatureName? If I understood the code correctly, instances of this class are passed as-is through formatting functions, this is not directly related to HTML escaping.